easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1252,6 +1252,8 @@ def process_easyconfig(path, build_specs=None, validate=True, parse_only=False,
                 dep = ec.toolchain.as_dict()
                 _log.debug("Adding toolchain %s as dependency for app %s." % (dep, name))
                 easyconfig['dependencies'].append(dep)
+                if dep['hidden']:
+                    easyconfig['hiddendependencies'].append(dep)
```

Comment:
@boegel: I tried various things and couldn't reproduce a situation where is needed. So, yes, let's drop it.